### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by rizakii**
Functionality : 

- Login & onboarding (fixed vendor thing ID)
- Display Onboarding result
- Display State (The  thing state is statically created after onboarding)
- Send Command with predefined scheme (Smart light)
- Create, Read, Update & Delete (CRUD) triggers

### CODE REVIEW

**Comment by rizakii** [comment]
@satoshikumano please review.

**Comment by satoshikumano** [comment]
README:
Oneliner would be easier.

```
npm install \ 
&& npm start
```

**Comment by satoshikumano** [comment]
LICENSE:

Can you apply to Apache 2?
(Let's apply Apache 2 for new open source projects.)

File: src/app/config.ts [hunk 1]
```diff
@@ -1,5 +1,13 @@
 
 var kii = require('kii-cloud-sdk').create();
-kii.Kii.initializeWithSite("___APPID___", "___APPKEY___", kii.KiiSite.US);
+/*
+US   : https://api.kii.com
+JP   : https://jp-api.kii.com
```
Comment (satoshikumano) [inline src/app/config.ts]: api-jp.kii.com

File: src/app/config.ts [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+SG   : https://sg-api.kii.com
```
Comment (satoshikumano) [inline src/app/config.ts]: api-sg.kii.com

File: src/app/config.ts [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+CN3  : https://cn3-api.kii.com
```
Comment (satoshikumano) [inline src/app/config.ts]: api-cn3.kii.com

File: src/app/config.ts [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+EU   : https://eu-api.kii.com
```
Comment (satoshikumano) [inline src/app/config.ts]: api-eu.kii.com

**Comment by satoshikumano** [comment]
On Server Code Trigger creation, there's choice 'Send Push notification' and 'Set Alarm'.
But I don't understand what's going on.

**Comment by satoshikumano** [comment]
Submitted issue #7 and #8.
Those are not mandatory for v1. Let's release it without them after fixed all review indications.

**Comment by rizakii** [comment]
@satoshikumano please check.

**Comment by satoshikumano** [comment]
LGTM.

**Pull request merged**